system-int-test/src/cmr/system_int_test/utils/index_util.clj

```diff
@@ -70,6 +70,41 @@
                     :throw-exceptions false})]
     (is (= 200 (:status response)) (:body response))))
 
+(defn reindex-concept-ignore-conflict-default
+  "Re-index concept without passing in ignore_conflict param should default to conflict being ignored."
+  [concept-id revision-id]
```

Comment:
I'd just have one function that can take optional query parameters in the util namespace. It should just call the function to reindex the concept with whatever parameters you want to use and return the response object. It wouldn't call `is`.

Then in your tests you would call `is` with what you are expecting to happen for each test to test that the status code matches what you expected.

I like that you are passing in the body of the response to `is` as the last parameter so that if there's a failure it prints out some extra info - we don't do that enough.

Reply:
+1